src/site/styles/layout.less

```diff
@@ -83,11 +83,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
+    .action-confirmation-overlay {
+        z-index: 200;
+        background: black;
+        position: fixed;
+        opacity: .5;
+        width: 100%;
+        height: 100%;
+        bottom: 0;
+        left: 0;
+        top: 0;
+        right: 0;
+    }
+
+    .action-confirmation-container {
+        position: fixed;
+        left: 0;
+        right: 0;
+        margin: -15% auto;
+        background: white;
+        z-index: 300;
+        width: 500px;
+        height: 200px;
```

Comment:
You're doing an awful lot of width and height setting. Try `display: flex` instead.